.github/workflows/fuzz-corpus.yml

```diff
@@ -0,0 +1,67 @@
+name: Check for new fuzzing samples
+on:
+  workflow_dispatch:
+  schedule:
+    # Run at 1 AM each day, so there is a `main`-branch baseline in the cache.
+    - cron: '0 1 * * *'
```

Comment:
I'd dial this back further.  Running weekly would be fine.

Keep in mind that you'll need to review and approve these PRs.